unit-test.spec.js

```diff
@@ -500,6 +500,18 @@ describe('lookup', function () {
     });
 
     describe("Charset won't be available if it is not mentioned", function () {
+        it("After an explicit charset", function () {
```

Comment:
Don't think this is part of the describe block.

Instead, add a new `Regression` describe block with these assertions (better title).
Also, add the related GitHub issue link as a comment.